core/modules/layout/layout.admin.inc

```diff
@@ -795,6 +795,19 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     );
   }
 
+  if (isset($form_state['build_info']['args'][0]->settings['additional_paths'])) {
+    $default_value = implode("\n", $form_state['build_info']['args'][0]->settings['additional_paths']);
+  }
+  else {
+    $default_value = '';
+  }
+  $form['additional_paths'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Additional paths'),
+    '#description' => t('Optionally specify one or more additional paths to be handled by this layout (in addition to the primary path above). These paths must already be provided by the system or another module. Use the percent symbol ("%") to indicate a placeholder in the path. e.g. "node/%" or "node/%/edit". The number of placeholders must be the same as the number of placeholders in the primary path.'),
```

Comment:
> Use the percent symbol ("%") to indicate a placeholder in the path. e.g. "node/%" or "node/%/edit".

This is already extensively explained in for the primary path above I don't think we need to do it again.
> The number of placeholders must be the same as the number of placeholders in the primary path.

Maybe we mean if additional path contains a placeholder it must be a child path of the primary path. Like "node/%/edit" is only allowed if the primary is "node/%". 

Otherwise this description sounds like if the primary is "node/%" then "user/%" is allowed since it's got the same number of placeholders.

Perhaps we could use a "show examples" details element like we do for the primary path since this field already seems to need  a lengthy description.

Reply:
Good ideas. I'm going to make an attempt at improving the help text.